packages/es-components/src/components/base/skeletonTile/SkeletonTile.md

```diff
@@ -0,0 +1,7 @@
+The skeleton component is used to represent something, typically a web component, is in the process of being loaded
+
+```jsx
+import Skeleton from './SkeletonTile.js';
```

Comment:
I can't help but feel like the documentation here is a bit misleading. The styling is heavily oriented towards the homepage tiles. It'll be harder to use this component as is outside this context.

Reply:
Agreed. I need to update that. When I started work I intended to make something more flexible, but the time cost just didn't seem worth it right now. I'll update that